### PR TITLE
Convert Footer to SitewideLinks

### LINKED ISSUE #1034: Create a SitewideLinks component
**Is your feature request related to a problem? Please describe.**
ServiceContext usage should be in containers/, not components/, and footer content should make use of this. 

**Describe the solution you'd like**
The SitewideLinks should replace the Footer component and have a flattened structure and its own Storybook story. SitewideLinks should be pulled in by the Footer container.

image from ios

**Is your feature request related to a problem? Please describe.**
ServiceContext usage should be in containers/, not components/.

**Describe alternatives you've considered**
N/A

- [x] Initially labelled with ["Refinement needed"](https://github.com/BBC-News/simorgh/labels/Refinement%20Needed) 

**Testing notes**
Regression testing for footer. This would be looking at the responsive style, current/future/older versions of web browsers and non-JS behaviour.

Dev insight: Just regression tests needed. Everything should look identical.

### PR DESCRIPTION
**Pull request by dr3**
Resolves #1034

Creates SitewideLinks component out of Footer

- [ ] Tests added for new features
- [x] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

TESTING NOTES

This splits up the footer component so that the brand and links sections are separate and no longer nested. This means the 'footer' component no longer exists inside storybook and instead a SitewideLinks component exists, which is just the links section with no 'footer' wrapper.

No other changes should be visible

### CODE REVIEW

File: src/app/components/SitewideLinks/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+## Description
+
+The `SitewideLinks` component is designed to be used at the bottom of an article, however can be used anywhere on a page. These links are generally intended to be to more general and legal BBC pages, but do not have to be. It also includes information about copyright. 
+
+## When to use this component
+
+It is currently used at the bottom of new BBC News and BBC News Persian article pages, however it has no markup requiring this to be its only application. It can be used anywhere on any page.
+
+## Considerations
+
```
Comment (ChrisBAshton) [inline src/app/components/SitewideLinks/README.md]: Blank?

**Review by ChrisBAshton — CHANGES_REQUESTED** [review]
This follows what we agreed https://github.com/BBC-News/simorgh/issues/1034#issue-386864715

BUT - is it too late to make a suggestion?

I don't see the value in a `SitewideLinks` component. We don't plan on reusing it in our own designs, and we wouldn't expect other teams to want to reuse it as a component. It's a collection of simple presentational components - I think it's best handled in the `FooterContainer`.

The less nesting, the better IMO. Thoughts?

**Review by ChrisBAshton — DISMISSED** [review]
Had a IRL chat with Drew, and now approve the concept of a SitewideLinks component, on the basis that:

1. This is _not_ just a component that pulls in a generic paragraph and a list. It's _very specifically styled_ to the footer, so is only useful in the sum of its parts, not its parts.
2. It does have value as a component in its own right (rather than living in our FooterContainer) as it could be used across lots of pages (not just article pages). Footers are generally the same across a site...

So fair enough.

Also nitpicked on the spelling of `SitewideLinks` vs `SiteWideLinks`, but ["sitewide" can count as a word in the same way as "website"](https://en.wiktionary.org/wiki/sitewide) so I'm happy to keep this as-is.

Approved! 🎉

**Review by ChrisBAshton — DISMISSED** [review]

**Comment by sareh** [comment]
Looks great! @dr3 Could you please add testing instructions in the PR description?  Explaining what part of Storybook stories have changed and the part of the page that James should look at regression testing when looking at this PR?

File: src/app/components/SitewideLinks/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+| links         | Array of Link objects | `[{href:'https://www.bbc.com', text: 'The BBC'}]` |
+| copyrightText | String                | `'Copyright BBC News'`                            |
+| externalLink  | Link object           | `{href:'https://www.bbc.com', text: 'The BBC'}`   |
+
+## When to use this component
+
+It is currently used at the bottom of new BBC News and BBC News Persian article pages, however it has no markup requiring this to be its only application. It can be used anywhere on any page.
+
+## Considerations
+### Landmark Roles
```
Comment (sareh) [inline src/app/components/SitewideLinks/README.md:20]: 👍 I like your explanation of usage, including the landmark contentinfo.

**Review by sareh — DISMISSED** [review]
👍 Looks great!

**Review by sareh — APPROVED** [review]

**Review by ChrisBAshton — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**